.clang-format

```diff
@@ -0,0 +1,9 @@
+---
+# Requires clang-format 10 or newer
+Language:        Cpp
+BasedOnStyle:  Google
+BraceWrapping:
+  AfterFunction:   true
```

Comment:
I assume we want `AfterMethod` or whatever the spelling is too? This is currently a weird middle ground

Reply:
Honestly I would just avoid overriding this one. It's making the diff a tiny bit larger, but avoiding the style exception is a clear win to me.

Reply:
Webkit only had `AfterFunction` set to `true`. I don't really care either way.